File: bestia-server-commons/src/test/java/net/bestia/server/AkkaClusterTest.java
package net.bestia.server;

import org.junit.Assert;
import org.junit.Test;


public class AkkaClusterTest {
	
	@Test(expected = NullPointerException.class)
	@SuppressWarnings("all")
	public void getNodeName_null_throws() {
		AkkaCluster.INSTANCE.getNodeName(null);
	}
	
	@Test
	public void getNodeName_ok() {
		String test = AkkaCluster.INSTANCE.getNodeName("test");
		Assert.assertEquals("/user/test", test);
		
		test = AkkaCluster.INSTANCE.getNodeName("test", "bla");
		Assert.assertEquals("/user/test/bla", test);
	}

}

File: bestia-server-commons/src/test/kotlin/bestia/server/AkkaClusterTest.kt
package bestia.server

import org.junit.Assert
import org.junit.Test

class AkkaClusterTest {

  @Test
  fun getNodeName_ok() {
    var test = AkkaCluster.getNodeName("test")
    Assert.assertEquals("/user/test", test)

    test = AkkaCluster.getNodeName("test", "bla")
    Assert.assertEquals("/user/test/bla", test)
  }
}
